spectool/jdtest.d.ts

```diff
@@ -0,0 +1,73 @@
+/// <reference path="jdspec.d.ts" />
+
+declare namespace jdtest {
+
+
+    interface ServiceTestMarkdownSpec {
+        /**
+         * associated service
+         */
+        classIdentifier: number;
+
+        /**
+         * Markdown source
+         */
+        source: string;
+    }
+
+    /**
+     * Service test.
+     */
+    interface ServiceTest {
+
+        description: string;
+
+        /**
+         * the service that this file targets
+         */
+
+        service: number;
```

Comment:
classIdentifier? or maybe serviceClassIdentifier?

Reply:
we use ``classIdentifier`` in the ServiceSpec.

Reply:
serviceClassIdentifier

Reply:
we use ``classIdentifier`` in ServiceSpec and ServiceMarkdownSpec already. ``serviceClassIdentifier`` is new.

Reply:
true, but if `classIdentifier` is in `Service` interface, it's clear it's the service class that we're talking about. here not so much

Reply:
It is called "ServiceTest" after all..